cmd/gitops-server/cmd/cmd.go

```diff
@@ -269,6 +285,34 @@ func runCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func listenAndServe(srv *http.Server, options Options) error {
+	if options.Insecure {
+		log.Println("TLS connections disabled")
+		return srv.ListenAndServe()
+	}
+
```

Comment:
Given insecure is false here we should probably check that the `options.TLSCertFile` and `options.TLSKeyFile` are not empty. If they are empty then we should return an error.

Reply:
And check that the files exist?

Reply:
@richardcase You are right we should check that those flags are not empty. On the other hand, if the files do not exist we will get this type of errors already:
<img width="635" alt="image" src="https://user-images.githubusercontent.com/3892786/157499065-0c3f2cf4-7a67-4846-ad5c-a703108846c0.png">
<img width="685" alt="image" src="https://user-images.githubusercontent.com/3892786/157499199-dba2506f-2e10-4def-b58e-52b6774b33cf.png">